### DISCUSSION TITLE
drilling down to a field on the form

### DISCUSSION THREAD

**Discussion by ravelab** [rsuite/rsuite#3191] [Q&A]
Hi, I am wondering if Form supports getting or setting a nested field? I tried something like this, it doesn't work. How do people workaround it?

<Form.Control name="list[0].field" {...rest} />

**Comment by SevenOutman** [comment]
Hi @ravelab, Form.Control does not support path syntax yet. For now you need to write a custom accepter component that handles the array value of `list`.

```jsx
function ListInput(props) { ... }

<Form.Control name="list" accepter={ListInput} {...rest} />
```

Here's an example from our docs site https://rsuitejs.com/components/form-validation/#dynamic-form-validation
